packages/myst-to-tex/src/index.ts

```diff
@@ -150,7 +150,7 @@ const handlers: Record<string, Handler> = {
     state.renderInlineEnvironment(node, 'sout');
   },
   break(node, state) {
-    state.write('\\\\');
+    state.write('\\newline');
```

Comment:
```suggestion
    // Use \newline instead of `\\` for breaks in LaTeX, it works in all phrasing contexts.
    // `\\` is used in tables to denote a new 
    state.write('\\newline');
```